tools/requests_gun/internal/request_maker/request_maker.go

```diff
@@ -74,6 +74,7 @@ func NewHttp(host string, port uint) *HttpRequestMaker {
 		host = "http://" + host
 	}
 	transport := http.DefaultTransport.(*http.Transport).Clone()
+	transport.DisableKeepAlives = true
```

Comment:
Was this change intended or a side-effect of some recent investigation that sneaked in by mistake?

Reply:
Yeah, this was intended. As I understand the http library keeps connections alive to reuse them, but we need to avoid this to simulate multiple clients making requests.